README.md

```diff
@@ -72,21 +42,37 @@
+[![codebeat badge](https://codebeat.co/badges/b2d1ce87-848b-440e-9d7e-df9883c0cd93)](https://codebeat.co/projects/github-com-vuetifyjs-vuetify-dev)
+[![Downloads](https://img.shields.io/npm/dm/vuetify.svg)](https://www.npmjs.com/package/vuetify)
+[![License](https://img.shields.io/npm/l/vuetify.svg)](https://www.npmjs.com/package/vuetify)
+[![zenhub](https://img.shields.io/badge/Managed_with-ZenHub-5e60ba.svg)](https://app.zenhub.com/workspace/o/vuetifyjs/vuetify/boards)
+[![Chat](https://img.shields.io/badge/chat-on%20discord-7289da.svg)](https://chat.vuetifyjs.com)
+[![Version](https://img.shields.io/npm/v/vuetify.svg)](https://www.npmjs.com/package/vuetify)
+[![CDN](https://img.shields.io/cdnjs/v/vuetify.svg)](https://cdnjs.com/libraries/vuetify)
 
-<p>Vuetify is a semantic component framework for Vue. It aims to provide clean, semantic and reusable components that make building your application a breeze.</p>
+## Introduction
 
-<p>Build <i>amazing</i> applications with the power of Vue and Material Design and a massive library of beautifully crafted components. Created according to Google's <strong><a href="https://material.io/" target="_blank">Material Design Spec</a></strong>, Vuetify components feature an easy-to-remember semantic design that shifts remembering complex classes and markup, to type-as-you speak properties that have simple and clear names.</p>
+Vuetify is a semantic component framework for Vue. It aims to provide clean, semantic and reusable components that make building your application a breeze.
 
-<p>Harness the power of the <a href="https://chat.vuetifyjs.com">Vuetify community</a> and get help 24/7 from talented developers across the world, the dev team and the creator, John Leider. Become a <a href="https://www.patreon.com/vuetify">backer</a> and get access to dedicated support from the team.</p>
-  
-<p>Vuetify supports all <strong>modern browsers</strong>, including IE11 and Safari 9+. From mobile to laptop to desktop, you can rest assured that your application will work as expected. Interested in the bleeding edge? Try the vue-cli Webpack SSR (Server side rendered) template and build websites optimized for SEO.</p>
+Build *amazing* applications with the power of Vue and Material Design and a massive library of beautifully crafted components. Created according to Google's **<a href="https://material.io/" target="_blank">Material Design Spec</a>**, Vuetify components feature an easy-to-remember semantic design that shifts remembering complex classes and markup, to type-as-you speak properties that have simple and clear names.
 
-<h2>Demo and Documentation</h2>
+Harness the power of the [Vuetify community](https://chat.vuetifyjs.com) and get help 24/7 from talented developers across the world, the dev team and the creator, John Leider. Become a [backer](https://www.patreon.com/vuetify) and get access to dedicated support from the team.
```

Comment:
Suggestion:

Harness the power of the [Vuetify community](https://chat.vuetifyjs.com) and get help 24/7 from the development team and our talented community members across the world.

Reply:
I updated the first phrase as your suggestion and I kept the second one.